packages/web-app-files/src/components/FilesList/ContextActions.vue

```diff
@@ -178,6 +179,9 @@ export default {
   methods: {
     getSectionClasses(index) {
       const classes = []
+      if (this.menuSections.length === 1) {
```

Comment:
By now I'd prefer a restructuring of all the different conditions into 0) early return, 1) handling of the paddings and 2) handling of the borders. If think what we want to have is:

## 0) early return
```js
if (!this.menuSections.length) {
  return classes
}
```

## 1) handling of paddings
```js
if (index < this.menuSections.length - 1) {
  classes.push('oc-pb-s')
}
if (index > 0) {
  classes.push('oc-pt-s')
}
```

Which means that all sections except the last get a bottom padding and all sections except the first get a top padding. For all sections in between the first and last sections this essentially boils down to `oc-py-s` but with less special case handling logic.

## 2) handling of borders
```js
if (index < this.menuSections.length - 1) {
  classes.push('oc-files-context-actions-border')
}
```

Which adds the bottom border for visual separation of sections to all sections except the last (and skips it entirely if you only have 1 section).